prusti-tests/tests/verify/fail/invariants/invariants-basic-2.rs

```diff
@@ -0,0 +1,25 @@
+extern crate prusti_contracts;
```

Comment:
Can all the invariants tests be placed into `verify_overflow`? Also have you looked if there are any `invariant`-related tests in `prusti-tests/tests_old`?